### PR TITLE
/sent endpoint completed

### PR DESCRIPTION
**Pull request by YoginiTayade**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-CRED_URL = {{ credentials.CRED_URL }}
-DID_URL = {{ credentials.DID_URL }}
-SCHEMA_URL = {{ credentials.SCHEMA_URL }}
-KEYCLOAK_REALM_ID= {{ credentials.KEYCLOAK_REALM_ID }}
-KEYCLOAK_CLIENT_ID= {{ credentials.KEYCLOAK_CLIENT_ID }}
-KEYCLOAK_CLIENT_SECRET= {{ credentials.KEYCLOAK_CLIENT_SECRET }}
-KEYCLOAK_URL= {{ credentials.KEYCLOAK_URL }}
-REGISTRY_URL= {{ credentials.REGISTRY_URL }}
-TELEMETRY_URL= {{ credentials.TELEMETRY_URL }}
+TESTVAR= development environment
```
Comment (rushi-tekdi) [inline .env:1]: discard all changes in .env file and then commit code again

File: envs/dev.env [hunk 2]
```diff
@@ -0,0 +1,31 @@
+PORT = 3000
```
Comment (rushi-tekdi) [inline envs/dev.env:1]: @YoginiTayade  delete envs folder from code

File: package.json [hunk 3]
```diff
@@ -52,6 +52,7 @@
     "qr-image": "^3.2.0",
     "qrcode": "^1.5.3",
     "reflect-metadata": "^0.1.13",
+    "reload": "^3.2.1",
```
Comment (rushi-tekdi) [inline package.json:55]: @YoginiTayade  why this new package install? what is use of that?

File: src/app.module.ts [hunk 4]
```diff
@@ -21,6 +21,8 @@ import { UdiseService } from 'src/services/udise/udise.service';
 import { TelemetryService } from './services/telemetry/telemetry.service';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { UsersModule } from './services/users/users.module';
+import { ClaimAttestService } from './claimAttest/claimAttest.service';
```
Comment (rushi-tekdi) [inline src/app.module.ts:24]: @YoginiTayade remove service claimattestservice import in app.module only keep claimattestmodule in app. module

File: .env (hunk 1 above)
Comment (YoginiTayade) [reply .env:1]: resolved

File: envs/dev.env (hunk 2 above)
Comment (YoginiTayade) [reply envs/dev.env:1]: resolved

File: package.json (hunk 3 above)
Comment (YoginiTayade) [reply package.json:55]: resolved

File: src/app.module.ts (hunk 4 above)
Comment (YoginiTayade) [reply src/app.module.ts:24]: resolved

**Pull request merged**